### PR TITLE
Photo essay bugs

### PR DESCRIPTION
**Pull request by blongden73**
Fixing some bugs in the photoessay template where the article creates a huge space when placing elements after the grid of four images.

### CODE REVIEW

File: static/src/stylesheets/inline/article-photo-essay.scss [hunk 1]
```diff
@@ -152,3 +154,53 @@
...
+.from-content-api ul {
+    display: inline-block;
+    width: 98%;
+    margin-top: 0;
+    @include mq($until: mobileLandscape) {
+        margin-top: -$gs-baseline / 2;
+    }
+}
+
+.from-content-api ul > li {
```
Comment (SiAdcock) [inline static/src/stylesheets/inline/article-photo-essay.scss]: Can we move this up into the preceding block?

```
& > li {

}
```

File: static/src/stylesheets/inline/article-photo-essay.scss [hunk 2]
```diff
@@ -152,3 +154,53 @@
...
+    @include fs-textSans(3);
+    color: #005689;
+    font-size: 12px;
+    width: 100%;
+    float: left;
```
Comment (SiAdcock) [inline static/src/stylesheets/inline/article-photo-essay.scss]: Is `float: left` and `width: 100%` necessary?

File: static/src/stylesheets/inline/article-photo-essay.scss [hunk 3]
```diff
@@ -152,3 +154,53 @@
...
+    }
+
+    @include mq($until: mobileLandscape) {
+        width: 98%;
+    }
+}
+
+.element--immersive + ul, .element--showcase + ul {
+    width: 150px;
+    margin-left: -170px;
```
Comment (SiAdcock) [inline static/src/stylesheets/inline/article-photo-essay.scss]: Can we comment this witchcraft?

File: static/src/stylesheets/inline/article-photo-essay.scss [hunk 4]
```diff
@@ -80,6 +79,9 @@
         }
 
         .section-title {
+            background-color: rgba(0, 0, 0, 1);
+            background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 1));
+
```
Comment (gustavpursche) [inline static/src/stylesheets/inline/article-photo-essay.scss]: I'd rather write it like

```css
background-color: rgb(0, 0, 0);
background: linear-gradient(transparent, rgb(0, 0, 0));
```

**Comment by SiAdcock** [comment]
Code is cool, man, I wish we didn't have to use so many tag and sibling selectors. Is this because the HTML is coming from CAPI and doesn't have any nice classes we can hook into?

File: static/src/stylesheets/inline/article-photo-essay.scss [hunk 5]
```diff
@@ -152,3 +154,53 @@
...
+    width: 98%;
+    margin-top: 0;
+    @include mq($until: mobileLandscape) {
+        margin-top: -$gs-baseline / 2;
+    }
+}
+
+.from-content-api ul > li {
+    @include fs-textSans(3);
+    color: #005689;
```
Comment (gustavpursche) [inline static/src/stylesheets/inline/article-photo-essay.scss]: `$guardian-brand` is the color, you are looking for :)

File: static/src/stylesheets/inline/article-photo-essay.scss [hunk 6]
```diff
@@ -152,3 +154,53 @@
...
+    font-size: 12px;
+    width: 100%;
+    float: left;
+    margin-right: 1%;
+    line-height: 14px;
+    padding-top: $gs-baseline / 2;
+    border-top: 1px solid #005689;
```
Comment (gustavpursche) [inline static/src/stylesheets/inline/article-photo-essay.scss]: `$guardian-brand` is the color, you are looking for :)

**Review by gu-stav — CHANGES_REQUESTED** [review]

**Review by SiAdcock — APPROVED** [review]
Nice, man 👍

**Review by gu-stav — APPROVED** [review]
Beautiful like a 🌷. Thanks for addressing everything!

**Pull request merged**